db/db.go

```diff
@@ -2,14 +2,15 @@ package db
 
 import (
 	"database/sql"
-	"encoding/hex"
-	"encoding/json"
-	"errors"
+	"encoding/base64"
 	"fmt"
 	"math/big"
 	"time"
 
-	"github.com/renproject/darknode/abi"
+	"github.com/renproject/darknode/tx"
+	"github.com/renproject/darknode/txengine"
+	"github.com/renproject/multichain/api/utxo"
+	"github.com/renproject/pack"
 )
 
 type TxStatus int8
```

Comment:
Is there a reason we do not use `tx.Status`?

Reply:
That type didn't exist when this package was created. I've updated it now